ibm_cloud_sdk_core/api_exception.py

```diff
@@ -13,17 +13,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import requests
+from typing import Optional
+
 
 class ApiException(Exception):
-    """
-    Custom exception class for errors returned from APIs.
+    """Custom exception class for errors returned from operations.
 
-    :param int code: The HTTP status code returned.
-    :param str message: A message describing the error.
-    :param dict info: A dictionary of additional information about the error.
-    :param response http_response: response
+    Args:
+        code: HTTP status code of the error response.
```

Comment:
No specifying type here?

Reply:
AFAIK you do not have to specify type for args as long as it is hinted.